### ISSUE TITLE
M300 commands being passed thru to printer on OctoPrint 1.9

### ISSUE THREAD

**Issue by andritolion** [stealthmonkey99/OctoPrint-PWMBuzzer#23]
On OctoPrint 1.9.x and newer, I've noticed that M300 commands created by this plugin are being passed through to the printer's speaker. My printer, the Ender 3v2, does not support different tones of beeps and thus my printer just makes one long and annoying beep when an event occurs, drowning out the Raspberry Pi's beeper.

**Comment by stealthmonkey99** [comment]
Hmm, does the "Suppress M300 command passthrough" option in the settings not work properly?  It should be preventing the command from being sent to the printer 🤔

**Comment by andritolion** [comment]
Yes, the option was enabled. For troubleshooting however, I toggled it off and back on again and it seems to have stopped passing through the print complete sound I set as of now.

**Comment by andritolion** [comment]
This does not seem to persist across a restart or happens randomly. Just completed a print and got a blast of beeps from the printer instead of the melody from the beeper.
